staging.inventory

```diff
@@ -58,6 +58,9 @@ vagrant-as-02
 hbase-master-node
 hbase-regionserver-nodes
 
+[oozie-node]
+vagrant-as-01
+
```

Comment:
what is the configuration here? on the namenode? the datanode? secondary namenode? this doesnt give any context and should probably be

```
[oozie-node:children]
hadoop-namenode-node
```
or something similar